Starts docs and exercises, implement verify-exercises (#3)
exercism/roc: docs/INSTALLATION.md

Reviewer: It's generally a great idea to link to the official install directions, but it's unclear what's the minimum needed to work offline on Exercism exercises so we could use a little elaboration in this file. https://www.roc-lang.org/install/linux_x86_64 covers how to install Roc but also Roc platform dependencies. As a student, do I just need the CLI? The REPL? The Rust, Zig, or C compilers at the bottom?

Author: You're right, I'll clarify this. Students just need to install Roc, it comes with a REPL, and a testing system.  Unless we add exercises to build your own Zig/C/Rust bindings, I don't think we'll need those installed (and I guess there could be specific installation instructions for these exercises if we ever add them).

# Installation

> To install Roc, please follow the instructions at [roc-lang.org/install](https://roc-lang.org/install). You only need to install Roc, and optionally the editor Extensions/Plugins of your choice, you don't need to install Rust or Zig (which are used to build Roc itself).

Reviewer: ```suggestion
To install Roc, please follow the instructions at [roc-lang.org/install](https://roc-lang.org/install).
You only need to install Roc, and optionally the editor Extensions/Plugins of your choice, you don't need to install Rust or Zig (which are used to build Roc itself).
```

# Installation

To install Roc, please follow the instructions at [roc-lang.org/install](https://roc-lang.org/install).

> You only need to install Roc, and optionally the editor Extensions/Plugins of your choice, you don't need to install Rust or Zig (which are used to build Roc itself).

Reviewer 2: Not sure if any student would have expected to install Rust of Zig, that seems more like an implementation detail.
Maybe instead of this:

```suggestion
There are also some [Editor Extensions/Plugins](https://www.roc-lang.org/install#editor-extensions) that you might want to install.
```

Or something like that

Author: Hi @ErikSchierboom ,
I'm sorry I missed your last comments and I merged the PR. I'm happy to create another one to address your comments if you want.
@keiravillekode or @BNAndras suggested above that it might be good to clarify exactly what needs to be installed, since Roc's installation instructions give you several options (including installing Zig or Rust, in case you want to build Roc from source or create some Roc platforms).

Reviewer 2: Ah right! Nevermind, forget about my comment then.
